docker-compose.dev.yml

```diff
@@ -4,12 +4,11 @@ services:
   minio_dev:
     image: minio/minio:latest
     ports:
-      - 9000:9000
+      - "${MINIO_PORT:-9000}:9000"
```

Comment:
THis Container is only for development and the Port is given as default 9000 in the application properties. No need to change it by env variable.

Reply:
I basically used the `prod-template.env` to list all our environment variables and the templates. The name is a bit misleading. You got any suggestions?

Reply:
Maybe an extra dev.env? We could then remove the defaults in theapplication properties and make it required and less prone for error caused by different values?